src/metabase/metabot/command.clj

```diff
@@ -107,27 +107,19 @@
 
 (defn- list-cards []
   (filter-metabot-readable
-   (let [collection-ids                  (metabot-visible-collection-ids)
-         root-collection-perms?          (contains? collection-ids nil)
-         other-visible-collection-ids    (filter some? collection-ids)
-
-         root-collection-filter-clause   (when root-collection-perms?
-                                           [:= :collection_id nil])
-         other-collections-filter-clause (when (seq other-visible-collection-ids)
-                                           [:in :collection_id other-visible-collection-ids])]
```

Comment:
I think I must have forgot to remove this when I put in the 

```clj
(collection/visible-collection-ids->honeysql-filter-clause (metabot-visible-collection-ids))
```

which accomplishes the same thing but more generally. good catch 👍